projects/cds/src/profiletag/model/profile-tag.model.ts

```diff
@@ -44,16 +44,11 @@ export enum PushEventNames {
   CART_CHANGED = 'CartSnapshot',
 }
 
-interface ProfiletagPushEvent {
-  name: PushEventNames;
+export interface ProfiletagPushEvent {
+  name: PushEventNames | string;
```

Comment:
Feel free to add your events names to `ProfileTagEventNames`

Reply:
I took this approach so that merchandising specific stuff doesn't leak into the profile tag code. Also it then means that the merchandising integration with the data layer provides a good example for customers if they wanted to integrate with it too, as everything is contained within the module integrating with the data layer rather than having to make changes to the data layer

I guess we could still allow it to be a string but also include the merch events in `ProfileTagEventNames` if you want?

Reply:
Yea, for extensibility you are right that we will want to allow strings. Just do what you think is cleanest for you guys. It may be good though to have one spot to see the default events -> and we could even change this to be 'CDSDefaultEvents', but feel free to do what you think is best